src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1819,6 +1819,11 @@ Copy_BibTeX_key_and_title=
 File_rename_failed_for_%0_entries.=
 Merged_BibTeX_source_code=
 Invalid_DOI\:_'%0'.=Ugyldig_DOI\:_'%0'.
+Invalid_IACR_identifier\:_'%0'.=
```

Comment:
Please try to use a more generic version in these strings. As of now, they are not reusable in other fetchers or situations. E.g. just use `invalid identifier` or replace `ICAR` by a parameter slot.